idaes/apps/grid_integration/tests/test_integration.py

```diff
@@ -0,0 +1,144 @@
+            return Path(pkg_file).parent
+
+    @pytest.mark.unit
+    def test_data_path_available(self, data_path: Path):
+        assert data_path.is_dir()
+
+    @pytest.fixture
+    def output_dir(self, tmp_path: Path) -> Path:
+        path = tmp_path / "DoubleLoop_output"
+        path.mkdir()
+        return path
+
+    @pytest.fixture
+    def plugin_path(self) -> Path:
+        with resources.path("idaes.apps.grid_integration.tests", "integration_test_plugin.py") as p:
+            return Path(p)
+
+    @pytest.mark.unit
+    def test_plugin_path_is_existing_file(self, plugin_path):
+        assert plugin_path.is_file()
+
+    @pytest.fixture
+    def prescient_options(self, data_path: Path, output_dir: Path, plugin_path: Path) -> PrescientOptions:
+        return {
+            "data_path": str(data_path),
+            "input_format": "rts-gmlc",
+            "simulate_out_of_sample": True,
+            "run_sced_with_persistent_forecast_errors": True,
+            "output_directory": str(output_dir),
+            "start_date": "07-10-2020",
+            "num_days": 2,
+            "sced_horizon": 4,
+            "ruc_mipgap": 0.01,
+            "reserve_factor": 0.0,
+            "deterministic_ruc_solver": "cbc",
+            "day_ahead_pricing": "LMP",
+            "symbolic_solver_labels": True,
+            "deterministic_ruc_solver_options": {
+                "feas": "off",
+                "DivingF": "on",
+            },
+            "sced_solver": "cbc",
+            "sced_frequency_minutes": 60,
+            "ruc_horizon": 48,
+            "compute_market_settlements": True,
+            "monitor_all_contingencies": False,
+            "output_solver_logs": False,
+            "price_threshold": 1000,
+            "contingency_price_threshold": 100,
+            "reserve_price_threshold": 5,
+            "plugin": {
+                "doubleloop": {
+                    "module": str(plugin_path),
+                    "bidding_generator": "10_STEAM",
+                }
+            },
+        }
+
+    @pytest.fixture
+    def run_simulator(self, prescient_options: PrescientOptions) -> None:
+        from prescient.simulator import Prescient
+
+        sim = Prescient()
+        sim.simulate(**prescient_options)
+
+    @pytest.fixture
+    def simulation_results_dir(self, run_simulator, output_dir):
+        return output_dir
+
+    @pytest.mark.unit
+    def test_output_dir_exist(self, simulation_results_dir):
+        assert os.path.isdir(simulation_results_dir)
+
+    @pytest.mark.component
+    def test_csv_files_saved(self, simulation_results_dir):
+
+        file_names = [
+            "hourly_gen_summary.csv",
+            "tracker_detail.csv",
+            "hourly_summary.csv",
+            "bus_detail.csv",
+            "overall_simulation_output.csv",
+            "virtual_detail.csv",
+            "bidding_model_detail.csv",
+            "bidder_detail.csv",
```

Comment:
Ideally we would check the output in this file, `tracker_detail.csv`, and `tracking_model_detail.csv`, but maybe for later.

Reply:
I will take a note on this and address it later.